### ISSUE TITLE
Center justify lost on some pages

### ISSUE THREAD

**Issue by jeff47** [pgaskin/kepubify#57]
I've noticed after using kepubify and comparing it to ePubs of the same book that some items that are center justified become left-justified.  For example, Book A has a centered picture below each chapter number in the ePub, but the picture becomes left justified in the kepub version.  In Book B, the copyright and dedication pages are similarly rendered. 

I can provide both files if you would like to look at them.

**Comment by pgaskin** [comment]
Can you email the EPUB to me so I can see what's causing this? I suspect it's due to styling applied to spans or divs, or possibly overly specific selectors.

**Comment by jeff47** [comment]
In the mail now.

**Comment by pgaskin** [comment]
Does the book render correctly if you just rename it to `.kepub.epub` rather than converting it?

**Comment by jeff47** [comment]
No, doesn't look like it - still not centered if I just rename the epub
without conversion.

**Comment by pgaskin** [comment]
OK, I'll look into it further tomorrow if I have time.

**Comment by yesimanoob** [comment]
The same issue occurs for me. I can send the epub source if necessary.

It would be much appreciated if this bug would be fixed.

**Comment by Hanpwny** [comment]
I know this post is from 5 year ago, but I'm having the same issue. It's specifically when a paragraph `<p>` follows an image `<img>` . It doesn't seem to be an html/css error because the epub renders correctly and calibre displays it fine.
